edgelet/hsm-rs/src/tpm.rs

```diff
@@ -171,7 +171,7 @@ impl SignWithTpm for Tpm {
 
 /// When buffer data is returned from TPM interface, it is placed in this struct.
 /// This is a buffer allocated by the C library.
-#[derive(Debug)]
+#[derive(Clone, Debug)]
```

Comment:
This should not be `Clone`. It does not clone the buffer so there'll be a double-free.